### PR TITLE
fix timezones on user-grows response

### PR DESCRIPTION
**Pull request by ice-cronus**
Accepts tz fields provided by FE and manage time in response to allow FE apply the timezone on it

### CODE REVIEW

File: cmd/eskimo/user_statistics.go [hunk 1]
```diff
@@ -80,7 +82,19 @@ func (s *service) GetUserGrowth( //nolint:gocritic // False negative.
...
-	result, err := s.usersRepository.GetUserGrowth(ctx, req.Data.Days)
+	tz := stdlibtime.UTC
+	if req.Data.TZ != "" {
+		var invertedTZ string
+		if req.Data.TZ[0] == '-' {
+			invertedTZ = "+" + req.Data.TZ[1:]
+		} else {
+			invertedTZ = "-" + req.Data.TZ[1:]
+		}
+		if t, err := stdlibtime.Parse("-07:00", invertedTZ); err == nil {
```
Comment (ice-ares) [inline cmd/eskimo/user_statistics.go:93]: Check how the balance history one is implemented

Comment (ice-ares) [reply cmd/eskimo/user_statistics.go:93]: And why is it inverted?

Comment (ice-cronus) [reply cmd/eskimo/user_statistics.go:93]: Cuz we need to substract it, so FE would be able to add it later. Or even better approach we could use - is not to reinvent a wheel here (trying to keep UTC response from the server), but passing dates in the desired TZ. Like:
```2023-07-07T23:59:59.99999Z+4:30```, if we're ok from the server, I'll discuss it with FE guys, but probably it won't be an issue there - datejs should handle that.

Comment (ice-ares) [reply cmd/eskimo/user_statistics.go:93]: Its bad either way, so its the same for me

**Pull request merged**